awslabs/aws-lambda-powertools-typescript
# test(common,logger,metrics): run e2e tests for metrics in both Node runtimes

## Description of your changes

Changes in this PR.
- refactor metrics e2e tests to make stack and resource name unique (so that they can be run in parallel)
- refactor metrics e2e tests to be consistent with logger and move common code to `commons`
- run e2e tests on both Node12 and Node14 runtimes

### How to verify this change

Run `npm run lerna-test:e2e` at the root.  

Every test should pass and metrics tests should be on both node14 and node12

### Related issues, RFCs
#497 

### PR status

***Is this ready for review?:*** NO  
***Is it a breaking change?:*** NO

## Checklist


### Breaking change checklist


---

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
